docs/release-notes/next/PT-epics-4177409.md

```diff
@@ -0,0 +1 @@
+* Deliver alpha version of AESTRATUM application implementing Stratum protocol for Aeternity node
```

Comment:
It would be nice enriching this with pointer to doc once available.

Reply:
I'll expand this once we have the doc finalized - next week.